### PR TITLE
Skip test_repartition_timeseries

### PR DESCRIPTION
**Pull request by mrocklin**
This test is absurdly slow and currently xfailed.
Lets skip it instead to accelerate things.

### CODE REVIEW

**Comment by mrocklin** [comment]
rerun tests

**Comment by mrocklin** [comment]
rerun tests

**Comment by mrocklin** [comment]
rerun tests

File: dask_cudf/tests/test_groupby.py [hunk 1]
```diff
@@ -15,7 +16,6 @@
         lambda df: df.groupby("x").count(),
         lambda df: df.groupby("x").min(),
         lambda df: df.groupby("x").max(),
-        lambda df: df.groupby("x").y.sum(),
         pytest.param(
             lambda df: df.groupby("x").y.agg(["sum", "max"]), marks=pytest.mark.xfail
         ),
```
Comment (mrocklin) [inline dask_cudf/tests/test_groupby.py]: @quasiben see this xfail approach here.

Comment (quasiben) [reply dask_cudf/tests/test_groupby.py]: 👍

**Comment by quasiben** [comment]
test: dask_cudf/tests/test_groupby.py::test_groupby[<lambda>6] seems to take a long time even though it is `xfailed`

**Comment by quasiben** [comment]
I switched the tests to `skip` instead of `xfail`

File: dask_cudf/tests/test_groupby.py [hunk 2]
```diff
@@ -15,29 +16,31 @@
...
     pdf = pd.DataFrame(
         {"x": np.random.randint(0, 5, size=10000), "y": np.random.normal(size=10000)}
     )
-
     gdf = cudf.DataFrame.from_pandas(pdf)
 
     ddf = dask_cudf.from_cudf(gdf, npartitions=5)
 
     a = func(gdf).to_pandas()
+    print(a)
```
Comment (mrocklin) [inline dask_cudf/tests/test_groupby.py]: Errant debug prints?

Comment (quasiben) [reply dask_cudf/tests/test_groupby.py]: thanks

**Pull request merged**